membership-attribute-service/app/models/AccountDetails.scala

```diff
@@ -153,9 +153,11 @@ object AccountDetails {
           "shouldDisplayEmail" -> selfServiceCancellation.shouldDisplayEmail,
           "phoneRegionsToDisplay" -> selfServiceCancellation.phoneRegionsToDisplay,
         ),
+        "billingCountry" -> billingCountry.iterator.mkString,
```

Comment:
I think this will rely on `toString` won't it? Might be better to explicitly state which field we want eg.
```scala
"billingCountry" -> billingCountry.map(_.alpha2).getOrElse(""),
```